### ISSUE TITLE
Longitude lost focus

### ISSUE THREAD

**Issue by JoeCohen** [MushroomObserver/mushroom-observer#2365]
Focus jumps from Longitude to Localities each time you change anything in Longitude.
Example:
- Edit an Observation whose Longitude has several (>=3) decimal digits.
- Click in the end of the Longitude field (to the right of the numbers)
- Back space (delete) one digit at a time.
Depending how fast you do this, the Localities field will auto update to match the updated Longitude and focus jumps to the start of the Localities string.
Repeat until you've erase the decimal point (and focus ups to the end of Localities.)
Now try to restore Longitude to multiple decimal digits. Painful, because each time you enter anything in Longitude, focus shifts to Localities.

**Comment by nimmolo** [comment]
Yeah, i've experienced this. It's a little mind boggling to fix. What needs to happen is something like the "keep button", or a "lock" icon on the Locality input. 

It is tricky differentiating between someone who wants to move the point of the obs and potentially change the locality, and someone who is futzing with their lat/lng.

What's the actual use case, though? (Not saying there isn't one, just that i couldn't think of one...) Why would people be trying to tweak the lat/lng numbers instead of clicking on the map? If people have a lat/lng they're trying to enter manually, it should work if they "Clear location", but the easiest way to adjust a point is by clicking a different place on the map.

**Comment by JoeCohen** [comment]
@nimmolo: I noticed it because I needed to both add a lat/lng and correct the Location for one of my Observations.
I forget the exact sequence of events an keyboard presses. It was a struggle to get things exactly right.
But if it doesn't come up that much, it's a lower priority. We all have bigger fish to fry.

**Comment by nimmolo** [comment]
Ok. I'm sure others will encounter it too, but for the record the easier path when you're doing that is to hit the "Clear location" button, add the lat/lngs, and then pick a locality. 

As long as one lat/lng field is blank, it will leave you alone until you fill both. You have to be quick with the second one, though, because as soon as it thinks it has a viable lat/lng pair of any number of digits, it starts looking for a Locality after a one-second delay.

We could just try to increase the Location-search delay, huh? What do you think

**Comment by JoeCohen** [comment]
I don't like longer delay.
Maybe add something to the instructions.
